### ISSUE TITLE
Server: Use Express Router to split up endpoints

### ISSUE THREAD

**Issue by prvnbist** [prvnbist/snippify#1]
The `index.js` file in `server` currently holds all the endpoints in the single file. Express `Router` method brings in the capability to split up endpoints into routes.

If you don't have experience with Express Router, go through this article - [Express Router](https://scotch.io/tutorials/learn-to-use-the-new-router-in-expressjs-4).

Go through the contribution guide in the root `README.md` file.

Steps to implement the Express Router functionlaity - 
- Go to the `server` folder and create `routes` folder.
- Inside `routes` folder, create two files namely - `snippet.js` & `label.js`
- Define the routes appropriate for `snippets` & `labels` into their own files.
- And finally import them into the `index.js` file and voila you're done.
- Do test the endpoints after you've done your magic.

Folder Structure post router implementation
```
snippify
- client
- server
  - routes
     - snippet.js
     - label.js
  - index.js
```

**Comment by dimglyn** [comment]
Hey! I would like to give this a try

**Comment by prvnbist** [comment]
Ok, I'll assign it you. Let me know if you need to ask anything.

**Comment by dimglyn** [comment]
I get some errors because app cannot find snippets folder. Is there any additional file I need to add? I thought I made a mistake with the path at first but it seems to happen at previous commits too.

**Comment by prvnbist** [comment]
As of now I'm just creating a snippets folder along side server & client folder. Later when I add electron to it, the user will be able to select the save location.

Could you tell what triggered the error?

**Comment by soniaabhyankar** [comment]
Hey! I would like to try this too.

**Comment by mora50** [comment]
I would like to try.

**Comment by prvnbist** [comment]
Hey @soniaabhyankar & @mora50 since this issue was initially assigned to @dimGlyn. I'm gonna wait another 6 hours before re-assigning it.

**Comment by mora50** [comment]
> Hey @soniaabhyankar & @mora50 since this issue was initially assigned to @dimGlyn. I'm gonna wait another day before re-assigning it.

Sorry @prvnbist , I worked on this issue later night, in a hackathon. I'm new on GitHub and didn't know that need to assign to issue the codes. But if you want to see my code, I'd be grateful

**Comment by prvnbist** [comment]
It's totally fine, I had to assign it to the first person that asks to resolve it so that multiple people don't work on same things simultaneously. 

Don't worry even if your PR doesn't get merged, I'll create few more issues and you can resolve those.

**Comment by prvnbist** [comment]
Resolved in #2
